packages/app/src/app/store/actions.js

```diff
@@ -1,10 +1,18 @@
 import axios from 'axios';
-
+import { Dictionary } from '@cerebral/fluent';
```

Comment:
What does `Dictionary` do?

Reply:
Dictionary is the same as `ObservableMap`... it basically IS an `ObservableMap`. Using `mobx-state-tree` this is managed "under the hood" because you use the type: `types.map`, but now we explicitly create them. They are observable dynamic maps... basically objects where you add/remove keys.